src/main/scala/cromwell/engine/backend/BackendCall.scala

```diff
@@ -0,0 +1,16 @@
+package cromwell.engine.backend
+
+import cromwell.binding.values.WdlValue
+import cromwell.binding.{Call, WdlStandardLibraryFunctions, WorkflowDescriptor}
+
+import scala.util.Try
+
+trait BackendCall {
```

Comment:
Ok - now that I got it up in the IDE it's another case of getting Backend and BackendCall mixed up in the github diffs via that execute thing. 

I'm a pretty firm believer that if there's absolutely no way to express what you're trying to express in the scala type system that either there's something out there you need to learn or you're likely doing something totally bogus and should rethink your actions. This doesn't seem totally bogus so there's gotta be a way. I'll poke at it.